code/datums/uplink_item.dm

```diff
@@ -849,11 +849,11 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 /datum/uplink_item/stealthy_weapons/martialarts
 	name = "Martial Arts Scroll"
 	desc = "This scroll contains the secrets of an ancient martial arts technique. You will master unarmed combat, \
-			deflecting all ranged weapon fire, but you also refuse to use dishonorable ranged weaponry. \
+			deflecting ranged weapon fire when we are in a defensive stance, but you also refuse to use dishonorable ranged weaponry. \
```

Comment:
```suggestion
			deflecting ranged weapon fire when you are in a defensive stance (throw mode), but you will also refuse to use dishonorable ranged weaponry. \
```

Reply:
perhaps it would be better to make this clear from an OOC perspective?

Reply:
Good idea